packages/zarf-registry/chart/values.yaml

```diff
@@ -58,3 +58,24 @@ extraEnvVars: []
 ## Additional ENV variables to set
 # - name: REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY
 #   value: "/var/lib/example"
+
+extraVolumes: []
+## Additional volumes to the pod.
+# - name: custom-ca-bundle
+#   configMap:
+#     name: custom-ca-bundle
+
+extraVolumeMounts: []
+## Additional volumeMounts to the registry container.
+#  - mountPath: /etc/ssl/certs/
+#    name: custom-ca-bundle
+#    readOnly: true
+
+extraConfigMaps: []
+## Additional configMaps to create in the release namespace
+# - name: custom-ca-bundle
+#   data:
+#     ca-certificates.crt: |
+#       -----BEGIN CERTIFICATE-----
+#       ...
+#       -----END CERTIFICATE-----
```

Comment:
Unless there is another use case you had in mind for this it may be better / more user friendly to just accept the certificate bundle as text without providing direct access to the volumes / config maps, etc (which requires more k8s knowledge).  It may also be better to do it with a `subPath` that way the volume mount places the single file in the volume without removing the other CA certs that may be there.

Reply:
Sorry for the delayed response the REGISTRY_CA_BUNDLE variable is what I was asking for - on the subPath the other CAs are in nearly all cases likely not needed I just usually prefer to add rather than replace to provide more flexibility